server/views/components/personal-transactions/template.njk

```diff
@@ -29,107 +27,144 @@
         </a>
       </li>
     {% endfor %}
-    {% if (data.selected === 'damage-obligations') or (data.shouldShowDamageObligationsTab) %}
+    {% if (data.selected === 'damage-obligations') or data.shouldShowDamageObligationsTab %}
       <li class="govuk-tabs__list-item{% if data.selected == 'damage-obligations' %} govuk-tabs__list-item--selected{% endif %}">
         <a class="govuk-tabs__tab" href="/money/damage-obligations">Damage obligations</a>
       </li>
     {% endif %}
   </ul>
   <div class="govuk-tabs__panel">
-    {% if data.transactions %}
-      <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Current balance</h2>
-      {% if data.balance.error %}
-        <p class="govuk-!-font-size-24">
-          {{data.balance.error}}, <a href="{{ returnUrl }}" class="govuk-link">try again</a>
-        </p>
-      {% else %}
-        <p>{{ data.balance.amount }}</P>
+    <ul class="transaction__balances">
+      {% if data.balance and not data.balance.userNotification %}
+        <li>
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Current balance</h2>
+
+          <p>{{ data.balance.amount }}</P>
+        </li>
       {% endif %}
-      {% if data.transactions.error %}
-        <p class="govuk-!-font-size-24">
-          {{data.transactions.error}}, <a href="{{ returnUrl }}" class="govuk-link">try again</a>
-        </p>
-      {% else %}
-        {% from "govuk/components/select/macro.njk" import govukSelect %}
-        <div class="transactions__selection_wrapper">
-          <form method="GET">
-            <div class="transactions__selection__form">
-              <input type="hidden" name="accountType" value="{{ data.selected }}"/>
-              {{ govukSelect({
+      {% if data.pendingTransactions and not data.pendingTransactions.userNotification %}
+        <li>
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Total pending payments</h2>
+          <p>{{ data.pendingTransactions.total }}</P>
+        </li>
+      {% endif %}
+    </ul>
+    {% if data.balance and data.balance.userNotification %}
+      <p class="govuk-!-font-size-24">
+        {{data.balance.userNotification}}, <a href="{{ returnUrl }}" class="govuk-link">try again</a>
+      </p>
+    {% endif %}
+    {% if data.pendingTransactions and data.pendingTransactions.userNotification %}
+      <p class="govuk-!-font-size-24">
+        {{data.pendingTransactions.userNotification}}, <a href="{{ returnUrl }}" class="govuk-link">try again</a>
+      </p>
+    {% elif data.pendingTransactions.rows %}
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Pending payments</h2>
+      <p>These are payments you have received but have not yet cleared. Pending payments do not show in your current balance</p>
+      <table id="pending-transactions" class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
```

Comment:
the table macro would have been quite nice for this (unless there's something weird going on which means you couldn't have used it)

Reply:
I don't there was an explicit reason, so I've updated the code to use GovUK tables instead